### PR TITLE
[script][common-items.lic] - Update to support new Steelclaw Clan bin

### PR DESCRIPTION
**Pull request by MazMedias**

### CODE REVIEW

File: common-items.lic [hunk 1]
```diff
@@ -356,6 +356,7 @@ module DRCI
 
     trashcans = DRRoom.room_objs
                       .reject { |obj| obj =~ /azure \w+ tree/ }
+                      .map { |obj| obj.gsub('small bin concealed with some nearby brush', 'small bin') }
```
Comment (rpherbig) [inline common-items.lic:359]: Does `DRC.get_noun` not work on "small bin concealed with some nearby brush"?

Comment (MazMedias) [reply common-items.lic:359]: It returns `concealed` so I implemented a workaround suggestion form @asechrest

Comment (asechrest) [reply common-items.lic:359]: Confirming it doesn't pare it down to the correct noun. (Interestingly, @KatoakDR 's noun regex refactor works perfectly on it, but I don't know that it's ready for release).

We threw this around in Discord quite a bit today. I suggested this solution as a way to just target this specific bin and use `gsub` to get it into a form where the noun regex would work on it, and then the rest of it flows fine.

I welcome better approaches, though.

Also appreciate @MazMedias submitting the PR!

Comment (KatoakDR) [reply common-items.lic:359]: I think I have a fix for the current noun parsing regex at https://github.com/rpherbig/dr-scripts/pull/5827

Interim fix until I can spend more time to get https://github.com/rpherbig/dr-scripts/pull/5435 ready for primetime

**Comment by rpherbig** [comment]
Overcome by #5827

**Comment by MazMedias** [comment]
It is working fine as far as I can tell. I believe the change to 372 still needs to be added to make it see 'small bin' rather than just 'bin,' but the current behavior leads to 'put my X in bin' which technically works just fine.

**Pull request closed without merging**